Library/Homebrew/extend/os/linux/os.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "os/linux/glibc"
+
+module OS
+  def self.family
+    :linux
+  end
+
+  def self.build_system_info
```

Comment:
Thoughts on this living on `DevelopmentTools`? Sorry to keep moving stuff around.

Reply:
I don't mind - it's hard to categorise. In terms of in terms of GCC version, it makes sense. In terms of the kernel or glibc version, less so. Xcode & CLT also lives in the OS module at the moment.

Reply:
Yeh, let's have this live in `DevelopmentTools` where it can call call out to Xcode/CLT for macOS and GCC/Glibc for Linux.